v1/message.go

```diff
@@ -37,6 +37,8 @@ type Content struct {
 	ChatID     string `json:"chat_id"`
 	Text       string `json:"text"`
 	ResponseTo string `json:"response-to"`
+	// The ENS name of the sender
+	Name string `json:"name"`
```

Comment:
```suggestion
	Name string `json:"name"` // the ENS name of the sender
```
so short that can be inlined :)